lib/about_yml/about.rb

```diff
@@ -36,16 +36,28 @@ def self.fetch_file_contents(client, repo_full_name)
       SafeYAML.load Base64.decode64(about['content'])
     end
 
+    def self.write_error(err, repo, result)
+      $stderr.puts('Error while parsing .about.yml for ' \
+        "#{repo.full_name}:\n #{err}")
+      result['errors'] << { repo.full_name => err.message }
+    end
+
+    def self.add_github_metadata(result, repo)
+      result['github'] = {
+        'name' => repo.full_name,
+        'description' => repo.description,
+      }
+    end
+
     def self.collect_repository_data(repo, client, result)
       collection = (repo.private == true) ? 'private' : 'public'
       repo_name = repo.full_name
       result[collection][repo_name] = fetch_file_contents client, repo_name
+      add_github_metadata result[collection][repo_name], repo
     rescue Octokit::NotFound
-      result['missing'] << repo.full_name
+      result['missing'] << repo.full_name unless collection == 'private'
```

Comment:
Actually, I think we still want to log which private repos don't have `.about.yml` files as well.